py/test/runbrick_test.py

```diff
@@ -414,14 +412,11 @@ def rbmain():
     assert(np.abs(src.pos.ra  - 244.77973) < 0.00001)
     # Results on travis vs local seem to differ?!
     assert(np.abs(src.pos.dec -  12.07234) < 0.00002)
-    #assert(np.abs(src.pos.dec -  12.07235) < 0.00001)
     src = cat[1]
     print('Source1', src)
-    assert(type(src) in [PointSource, GaiaSource])
-    assert(np.abs(src.pos.ra  - 244.77828) < 0.00002)
-    assert(np.abs(src.pos.dec -  12.07248) < 0.00001)
-    # DevGalaxy(pos=RaDecPos[244.77975494973529, 12.072348111713127], brightness=NanoMaggies: g=19.2, r=17.9, z=17.1, shape=re=2.09234, e1=-0.198453, e2=0.023652,
-    # PointSource(RaDecPos[244.77833280764278, 12.072521274981987], NanoMaggies: g=25, r=23, z=21.7)
+    assert(type(src) ==  PointSource)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using type() instead of isinstance() for a typecheck.](https://app.codacy.com/manual/dstndstn/legacypipe/pullRequest?prid=5336726)